bin/cli.js

```diff
@@ -78,6 +79,13 @@ program
   .action(config.wrap(program, init.cmd));
 
 program
+  .command("env")
+  .description("Output configured env variables")
+  .option("-s --site-id [id]", "Fetch from site with <id>")
+  .option("-f --file [filename]", "Save to file called <filename>")
```

Comment:
Unfortunately the site selection ui uses stdOut, so if I redirect the output without passing a site id I don't get the site selection ui in my terminal.
This should happen on stdErr, but since this would require large amounts of re-structuring, I solved it this way.